pkg/medusa/reconcile.go

```diff
@@ -160,6 +168,22 @@ func UpdateMedusaInitContainer(dcConfig *cassandra.DatacenterConfig, medusaSpec
 	}
 }
 
+func medusaInitContainerResources(medusaSpec *api.MedusaClusterTemplate) corev1.ResourceRequirements {
+	if medusaSpec.Resources != nil {
```

Comment:
Is this intended to be ``if medusaSpec.InitContainerResources != nil`` ?

Reply:
yep, good catch 👍